blades/abstract_blade.h

```diff
@@ -109,6 +111,47 @@ class AbstractBlade : public BladeBase, public SaberBase {
     return GetPrimaryBlade() == this;
   }
 
+  // Blaster Effects
+  void SB_Stun() override {
+    addEffect(EFFECT_STUN, (200 + random(700)) / 1000.0f);
```

Comment:
It doesn't make sense for the "location" of the stun effect to be random, so just make it zero I think. (Same for all of the effects below.)

Reply:
Will fix. I copied blast for these. Good catch.